src/components/FormSettingsUserData.tsx

```diff
@@ -0,0 +1,171 @@
+            <Figure>
+              {previewAvatar && (
+                <>
+                  <Figure.Image
+                    roundedCircle
+                    src={previewAvatar}
+                    alt="preview"
+                  />
+                </>
+              )}
+            </Figure>
+            <Figure>
+              <Form.Group as={Row} controlId="avatarFile" className="mb-3">
+                <div className="d-grid gap-2">
+                  <Form.Label>Upload user avatar</Form.Label>
+                  <Form.Control
+                    type="file"
+                    size="sm"
+                    ref={fileRefAvatar}
+                    onChange={handleAvatarUpload}
+                  />
+                </div>
+              </Form.Group>
+            </Figure>
+
+            {/*----user image -------*/}
+            <Figure>
+              {previewImage && (
+                <>
+                  <Figure.Image
+                    fluid
+                    rounded
+                    src={previewImage}
+                    alt="preview"
+                  />
+                </>
+              )}
+            </Figure>
+            <Figure>
+              <Form.Group controlId="bgFile" className="mb-3">
+                <div className="d-grid gap-2">
+                  <Form.Label>Upload user picture</Form.Label>
+                  <Form.Control
+                    type="file"
+                    size="sm"
+                    ref={fileRefImage}
+                    onChange={handleImageUpload}
+                  />
+                </div>
+              </Form.Group>
+            </Figure>
+
+            {/*----NameUser input -------*/}
+            <Form.Group className="mb-3" controlId="inputNameUser">
+              <Form.Control type="text" placeholder="Name" />
+            </Form.Group>
+            {/*----BioUser field -------*/}
+            <Form.Group className="mb-3" controlId="inputBioUser">
+              <Form.Control as="textarea" rows={2} placeholder="Bio" />
+            </Form.Group>
+            {/*----LocationUser input -------*/}
+            <Form.Group className="mb-3" controlId="inputLocationUser">
+              <Form.Control type="text" placeholder="Location" />
+            </Form.Group>
+            {/*----WebsiteUser input -------*/}
+            <Form.Group className="mb-3" controlId="inputWebsiteUser">
+              <Form.Control type="text" placeholder="Website" />
+            </Form.Group>
+            {/*----BirthDateUser input -------*/}
+            <Form.Group className="mb-3" controlId="inputBirthDateUser">
+              <Form.Control
+                type="date"
+                placeholder="Date"
+                data-date-format="YYYY/MM/DD"
+              />
+            </Form.Group>
+            {/*----advanced settings button -------*/}
+            <div className="d-grid gap-2">
+              <Button variant="light">Switch to professional</Button>
```

Comment:
Use n18i for the text. So your module could be swithed to another language.

Reply:
done